ansible/deploy.yml

```diff
@@ -217,6 +194,8 @@
     - role: activate-dashd-sporks
       when: dash_network != "mainnet" and dash_network != "testnet"
 
+# Wait for chainlocks here, then restart platform services?
```

Comment:
@shumkov Currently we need to run the tool twice and set a value for `INITIAL_CORE_CHAINLOCKED_HEIGHT` in order to start platform. If we put logic here to watch the network for chainlocks, add the value to config and restart drive so it sees the value.

Reply:
Actually another reason for a restart here is we need to retemplate and restart dashd with the new bls privkey from the previous `mn-init` task.

Reply:
We should set hardcoded 2100 blocks to config and even if chainlock happens later that's ok. 

Regarding mn init it worked before somehow. Not sure we need to change this logic.